jdisc-security-filters/src/main/java/com/yahoo/jdisc/http/filter/security/base/JsonSecurityRequestFilterBase.java

```diff
@@ -25,7 +26,7 @@
  *
  * @author bjorncs
  */
-public abstract class JsonSecurityRequestFilterBase extends AbstractComponent implements SecurityRequestFilter {
+public abstract class JsonSecurityRequestFilterBase extends ChainedComponent implements SecurityRequestFilter {
```

Comment:
It's sad that the next person has to figure out _their_ filter isn't a child of this class, too, but I don't see a good solution here. Do you?

Reply:
I'm not familiar with the framework. If it do require class inheritance I don't see any quick fix.